### ISSUE TITLE
Can't connect or split tunnel parsec

### ISSUE THREAD

**Issue by TheRealDadbeard** [mullvad/mullvadvpn-app#4235]
Windows 11 22h2

Mullvad 2022.5

### Issue description

When I am connected with Mullvad I can't connect at all via Parsec from outside. I tried split tunneling but that didn't work either. Only disconnecting let's me connect.

**Comment by EntranceJew** [comment]
I was able to connect to a machine running parsec by:
1) inside parsec's network setting, enable UPnP
2) enabling split tunneling
3) whitelisting `parsec.exe` and `parsecd.exe` manually
4) restart

on your client you shouldn't need to enable anything else.
my target machine was on my lan, so I'm not sure

**Comment by TheRealDadbeard** [comment]
Upnp is on. I can see the ports being opened and they are forwarded correctly.

I can connect over Lan with or without mullvad but never with split tunneling.

I can never connect over internet with mullvad connected. Try it this way

**Comment by EntranceJew** [comment]
I asked some people outside my network to attempt it and you are correct,
there's no configuration which allows them to connect to my machine over parsec with split tunneling.

It only works after routing the entire connection.

**Comment by TheRealDadbeard** [comment]
Yeah I hope this is fixable. I think parsec has some special network requirements that don't pass through the vpn.

I have teamviewer installed for this so I can toggle the vpn when I need parsec for now.

**Comment by Still34** [comment]
This has been an on-going issue for more than a year well before this issue was opened. It would be nice if this can somehow eventually be resolved.
